tensorflow_serving/servables/tensorflow/util.cc

```diff
@@ -47,6 +47,48 @@ auto* example_count_total = monitoring::Counter<1>::New(
     "/tensorflow/serving/request_example_count_total",
     "The total number of tensorflow.Examples.", "model");
 
+// Metrics by model
+auto* model_request_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_count",
+    "The total number of requests.", "model");
+
+auto* model_request_fail_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_fail_count",
+    "The total number of faled requests.", "model");
+
+auto* model_latency_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_latency_usec",
+    "The total time spent on executing graphs in microseconds.", "model");
+
+auto* model_latency_histogram = monitoring::Sampler<1>::New(
+    {"/tensorflow/serving/model_request_latency_histogram_usec",							    
+     "The total time spent on executing graphs in microseconds.", "model"},
+    // It would be nice to be able to set the parameters flexibly.
+    monitoring::Buckets::Explicit({
+	1000, 2000, 3000, 4000, 5000, 7000, 9000, 11000, 13000, 15000,
+	17000, 19000, 21000, 24000, 27000, 30000, 33000, 35000, 38000}));
+
+// All processing metrics
```

Comment:
metric aggregator/collector systems can sum things up based on per-model counters (above). no need to add `all_` counters.